tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/effects/debugger_effects.ts

```diff
@@ -653,6 +657,88 @@ export class DebuggerEffects {
     );
   }
 
+  /**
+   * Listens to graph-op focus events.
+   *
+   * Load graph op info from the /graphs/op_info route when necessary.
+   *
+   * @returns the IDs of the stack frames of the op's creation for downstream
+   * consumption.
+   */
+  private loadGraphOpInfo(): Observable<string[]> {
+    return this.actions$.pipe(
+      ofType(graphOpFocused),
+      withLatestFrom(
+        this.store.select(getActiveRunId),
+        this.store.select(getLoadingGraphOps)
+      ),
+      filter(([actionData, runId, loadingOps]) => {
+        const {graph_id, op_name} = actionData;
+        return (
+          runId !== null &&
+          (loadingOps[graph_id] === undefined ||
+            loadingOps[graph_id][op_name] === undefined ||
+            loadingOps[graph_id][op_name] === DataLoadState.FAILED)
```

Comment:
nit: perhaps use shorthand `!loadingOps[graph_id] || !loadingOps[graph_id][op_name]` ?

Can / will `loadingOps` ever store `DataLoadState.NOT_LOADED` ?

On a side note, I'm a little worried by how we use serializable objects as state to hold user values.  Someone can make an `op_name` called "toString" or "\__proto__".